Tools/Scripts/webkitpy/port/glib.py

```diff
@@ -129,6 +136,13 @@ def setup_environ_for_server(self, server_name=None):
 
         return environment
 
+    def setup_environ_for_minibrowser(self):
+        env = os.environ.copy()
+        env['WEBKIT_EXEC_PATH'] = self._build_path('bin')
+        env['WEBKIT_INJECTED_BUNDLE_PATH'] = self._build_path('lib')
+        env['LD_LIBRARY_PATH'] = self._get_path_value_updated_for_environ(env, 'LD_LIBRARY_PATH', self._build_path('lib'))
```

Comment:
```
env['LD_LIBRARY_PATH'] = self._prepend_to_env_value(self._build_path('lib'), env.get('LD_LIBRAY_PATH', ''))
```
Or as you wish to name it, and the logic in the helper function can get slightly simpler.

Reply:
Nice! will use it :)